src/matcha_ml/cli/cli.py

```diff
@@ -24,6 +25,13 @@
 
 app = typer.Typer(no_args_is_help=True, pretty_exceptions_show_locals=False)
 
+# Create a group for all subcommands for analytics command
+app.add_typer(
+    analytics.app,
+    name="analytics",
+    help="Enable or disable the collection anonymous of usage data (enabled by default).",
```

Comment:
"of" in the wrong place -> Enable or disable the collection of anonymous usage data (enabled by default)